spec-main/api-autoupdater-darwin-spec.ts

```diff
@@ -307,7 +307,7 @@ describeFn('autoUpdater behavior', function () {
 
         await relaunchPromise;
         expect(requests).to.have.lengthOf(3);
-        expect(requests[2]).to.have.property('url', '/update-check/updated/2.0.0');
+        expect(requests[2].url).to.equal('/update-check/updated/2.0.0');
```

Comment:
left side seems better..?

Reply:
It results in a very obscure error being thrown when that assertion fails (completely unrelated to the assertion).  Calling `inspect()` on a http request doesn't work and that's what the assertion library tries to log 😢